ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.js

```diff
@@ -0,0 +1,490 @@
+          return;
+
+        if (data.role === 'healer')
+          return output.tankBuster();
+
+        return output.avoidTankBuster();
+      },
+      run: (data) => delete data.busterTargets,
+      outputStrings: {
+        tankBuster: {
+          en: 'Tank Buster',
+          de: 'Tank buster',
+          fr: 'Tank buster',
+          ja: 'タンクバスター',
+          cn: '坦克死刑',
+          ko: '탱버',
+        },
+        avoidTankBuster: {
+          en: 'Avoid tank buster',
+          de: 'Tank buster ausweichen',
+          fr: 'Évitez le tank buster',
+          ja: 'タンクバスターを避ける',
+          cn: '远离坦克死刑',
+          ko: '탱버 피하기',
+        },
+      },
+    },
+    {
+      id: 'Paradigm Knave Magic Artillery Alpha Spread',
+      netRegex: NetRegexes.headMarker({ id: '00A9' }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: 4,
+      response: Responses.spread(),
+    },
+    {
+      id: 'Paradigm Knave Lunge',
+      netRegex: NetRegexes.startsUsing({ id: '5EB1', source: 'Knave of Hearts', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5EB1', source: 'Herzbube', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5EB1', source: 'Jack', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5EB1', source: 'ジャック', capture: false }),
+      durationSeconds: 5,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Knockback (from boss)',
+          de: 'Rückstoß (vom Boss)',
+        },
+      },
+    },
+    {
+      id: 'Paradigm Copied Knave Lunge',
+      netRegex: NetRegexes.startsUsing({ id: '5EB1', source: 'Copied Knave', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5EB1', source: 'Kopiert(?:e|er|es|en) Herzbube', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5EB1', source: 'Réplique De Jack', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5EB1', source: '複製サレタジャック', capture: false }),
+      condition: (data) => !data.cloneLunge,
+      durationSeconds: 5,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Knockback (from clone)',
+          de: 'Rückstoß (vom Klon)',
+        },
+      },
+      run: (data) => data.cloneLunge = true,
+    },
+    {
+      id: 'Paradigm Copied Knave Lunge Out of Middle',
+      netRegex: NetRegexes.startsUsing({ id: '60C8', source: 'Knave of Hearts', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '60C8', source: 'Herzbube', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '60C8', source: 'Jack', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '60C8', source: 'ジャック', capture: false }),
+      durationSeconds: 5,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Knockback (from clone) + Out of Middle',
```

Comment:
`Get Knocked Out Of Middle` or `Knockback -> Out Of Middle`?

Reply:
Do these knockbacks need/want to mention their origin (boss/clone)?

Reply:
I'm a little indifferent here, mostly because I expect followup triggers to say Knockback South or Knockback East etc.